pkg/interfaces/interfaces_test.go

```diff
@@ -81,7 +81,9 @@ func TestGetInterfaceIndices(t *testing.T) {
 		t.Fatal(err)
 	}
 	t.Logf("bond interface index %d", bond.Attrs().Index)
-	defer netlink.LinkDel(bond)
+	defer func() {
+		_ = netlink.LinkDel(bond)
+	}()
```

Comment:
If this is purely a stylistic change which continues to ignore the error from `LinkDel` is there actually any value?

Reply:
wanted to keep linter quiet but I will add check and err log here while I am here